components/search/SearchSuggestion.vue

```diff
@@ -370,25 +402,25 @@ const router = useRouter()
 const route = useRoute()
 const { $consola } = useNuxtApp()
 
-const updateSearchUrl = () => {
+const updateSearchUrl = (path: string) => {
   const { name } = props
   if (name) {
     router
-      .replace({
```

Comment:
I would keep `router.replace`, otherwise it would cause too much navigation history on the browser while searching.